protopipe_grid_interface/scripts/submit_jobs.py

```diff
@@ -610,11 +612,14 @@ def main():
         job = dirac.submitJob(j)
         if not job["OK"]:
             log.critical("Job submission failed: %s", job)
-            raise RuntimeError(f"Job submission failed: {job}")
```

Comment:
it is my experience that when a job submission fails, the cause is so "simple" that it affects any job in the same batch

I was raising an error becomes it doesn't make much sens to log every failed submission since it will be the same of each job

Reply:
I removed it because I had some errors where a single job failed to submit, in the middle of submitting 15000 jobs... So it was very annoying to have the rest also stop.  It was just a temporary network failure. So this way, I count how many fail, but keep going.  Of course, if there are many failures, maybe it should stop